sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/application-offering-change-request.aest.controller.ts

```diff
@@ -91,4 +100,19 @@ export class ApplicationOfferingChangeRequestAESTController extends BaseControll
       { hasAuditAndNoteDetails: true },
     );
   }
+
+  /**
+   * Updates an application offering change request.
+   * @param applicationOfferingChangeRequestId application offering change request id.
+   * @param payload information to update the application offering change request.
+   */
+  @Patch(":applicationOfferingChangeRequestId")
```

Comment:
As discussed, missing the ministry role here. 

The todo comment description can mention the role as well.